stats/cloud/config.go

```diff
@@ -47,6 +47,9 @@ type Config struct {
 	// The time interval between periodic API calls for sending samples to the cloud ingest service.
 	MetricPushInterval types.NullDuration `json:"metricPushInterval" envconfig:"K6_CLOUD_METRIC_PUSH_INTERVAL"`
 
+	// This is how many parallel pushes will be done at the same time to the cloud
+	MetricPushParallel null.Int `json:"metricPushParallel" envconfig:"K6_CLOUD_METRIC_PUSH_PARALLEL"`
```

Comment:
`MetricPushConcurrency` seems like a better name?

Reply:
fixed in 4632408